lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -1163,6 +1163,7 @@ NativeCodeGenerator::CheckCodeGen(Js::ScriptFunction * function)
         // it either corresponds with this codegen episode (identified by function->entryPointIndex) of the function body
         // or one that was scheduled after. In the latter case originalEntryPoint will get updated if and when
         // that last episode completes successfully.
+#ifdef ENABLE_SCRIPT_PROFILING
```

Comment:
As mentioned earlier- this is not necessary right now since Backend doesn't build